docs/BR.md

```diff
@@ -1357,9 +1353,7 @@ b. cRLDistributionPoints
 
 c. authorityInformationAccess
 
-    With the exception of stapling, which is noted below, this extension MUST be present. It MUST NOT be marked critical, and it MUST contain the HTTP URL of the Issuing CA's OCSP responder (accessMethod = 1.3.6.1.5.5.7.48.1). It SHOULD also contain the HTTP URL of the Issuing CA's certificate (accessMethod = 1.3.6.1.5.5.7.48.2).
-
-    The HTTP URL of the Issuing CA's OCSP responder MAY be omitted, provided that the Subscriber "staples" the OCSP response for the Certificate in its TLS handshakes [RFC4366].
+    This extension MUST be present. It MUST NOT be marked critical, and it MUST contain the HTTP URL of the Issuing CA's OCSP responder (accessMethod = 1.3.6.1.5.5.7.48.1). It SHOULD also contain the HTTP URL of the Issuing CA's certificate (accessMethod = 1.3.6.1.5.5.7.48.2).
```

Comment:
The problem with this section is that it applies to subordinate CAs, but the method it describes cannot be used to provide revocation status information for subordinate CAs. That's provided by RFC 6961, from Yngve Pettersen, but for which few (no?) browser members support.

Because of that, it's possible to meet this requirement using the language written, so it's removed. Support for status_request_v2 isn't high on most folks TLS1.2 agenda. In TLS1.3, the whole mechanism is changed (and implicitly includes this, although that may not be desirable). I suspect that this is probably a broader discussion for Browser Members as to whether to support this case in the future, but given that it cannot and is not supported now, it seemed OK to remove.